__tests__/OmnichannelChatSDK.spec.ts

```diff
@@ -1536,25 +1332,16 @@ describe('Omnichannel Chat SDK', () => {
 
             jest.spyOn(chatSDK.OCClient, 'sessionInit').mockResolvedValue(Promise.resolve());
 
-            await chatSDK.startChat(optionalParams);
-
-            const sessionInitOptionalParams = {
-                initContext: {
-                    preChatResponse: optionalParams.preChatResponse,
-                    browser: optionalParams.browser,
-                    os: optionalParams.os,
-                    locale: optionalParams.locale,
-                    device: optionalParams.device
-                }
-            }
+            const chatContext = await chatSDK.getCurrentLiveChatContext();
 
-            expect(chatSDK.OCClient.sessionInit.mock.calls[0][1]).toMatchObject(sessionInitOptionalParams);
+            expect(Object.keys(chatContext).length).toBe(0);
+            expect(Object.keys(chatContext).includes('chatToken')).toBe(false);
+            expect(Object.keys(chatContext).includes('requestId')).toBe(false);
         });
 
-        it('[LiveChatV1] ChatSDK.startChat() with initContext defined should override IStartChatOptionalParams', async() => {
```

Comment:
This test is valid. Its goal is validate the precedence of setting `initContext`. We need to remove the `v1` dependency